Libraries/LibWeb/Painting/BackgroundPainting.h

```diff
@@ -24,6 +24,7 @@ struct ResolvedBackgroundLayerData {
     CSSPixelRect image_rect;
     CSS::Repeat repeat_x;
     CSS::Repeat repeat_y;
+    CSS::MixBlendMode mix_blend_mode;
```

Comment:
Nit: Just call this `blend_mode`. Looks like `<blend-mode>` is the type name used in the spec currently.